src/types/domain/Commander.ts

```diff
@@ -3,8 +3,15 @@ export type Commander = {
     name: string;
     colorIdentity: string[];
     /**
-     * A collection of match IDs of matches that the commander is played in
+     * A collection of match IDs of matches that the commander is played in.
      */
     matches: string[];
+    /**
+     * Number of games we want to take into account when calculating stats (i.e. 4 player pods).
+     */
+    validMatchesCount: number;
+    /**
+     * The number of wins across all this player's VALID matches (i.e. 4 player pods).
```

Comment:
nit, instead of "ie". Can we be a bit verbose here?

"Valid matches are those determined by some minimum criteria. For example: having the correct number of players."